pages/options.js

```diff
@@ -234,5 +234,11 @@ var StravaEnhancementSuiteOptions = [
     , "description": "Show same-activity Flybys only (runs or rides) in the Flyby viewer."
     , "image": false
     , "default": false
+  }, {
+      "name": "show_kudo_all_button"
+    , "title": "Show button to give Kudos to all"
+    , "description": "Show button in the header bar to give Kudos to all displayed activities that need some love"
```

Comment:
Totally +1 with the idea, can you just round-off the "need some love" bit here and make it more object? :)

Reply:
I already had this button as a Tampermonkey script, figured this might be a good way to share :-D

Reply:
Haha, I have one in Tampermonkey for untappd.com ;)